Gopkg.toml

```diff
@@ -38,15 +10,48 @@ required = [
     "sigs.k8s.io/controller-runtime/pkg/source",
     "sigs.k8s.io/testing_frameworks/integration", # for integration testing
     "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
+    "github.com/openshift/cluster-autoscaler-operator/pkg/apis",
+    "github.com/openshift/api/config/v1",
+    "github.com/onsi/ginkgo",
+    "github.com/onsi/gomega",
 ]
 
-[[constraint]]
-  name = "github.com/golang/mock"
-  version = "1.2.0"
+[[override]]
+  name = "k8s.io/kube-aggregator"
+  version = "kubernetes-1.12.5"
+
+[[override]]
+  name = "github.com/openshift/cluster-api"
```

Comment:
We have just bumped to the latest cluster-api and k8s 1.13. Can you sync with that as well?

https://github.com/openshift/cluster-api-provider-aws/blob/master/Gopkg.toml#L31-L61